functions/Copy-DbaAgentAlert.ps1

```diff
@@ -155,7 +155,7 @@ function Copy-DbaAgentAlert {
                 $sourceAlertNotifications = Get-DbaAgentAlert -SqlInstance $sourceServer | Where-Object Name -eq $alertName | Select-Object Notifications
                 $sourceAlertOperators = $sourceAlertNotifications.Notifications.OperatorName
                 if ($null -ne $sourceAlertOperators) {
-                    $missingOperators = Compare-Object -ReferenceObject $sourceAlertOperators -DifferenceObject $destServerOperators | Where-Object {$_.sideIndicator -eq "<="} | Select-Object -expandproperty InputObject                
+                    $missingOperators = Compare-Object -ReferenceObject $sourceAlertOperators -DifferenceObject $destServerOperators | Where-Object {$_.sideIndicator -eq "<="} | Select-Object -expandproperty InputObject
```

Comment:
Use of external commands (even with built-in) require special attention because they will mess with our messaging system. (cause sea of red).

This alone is going to fail because you can't compare a populated object with an empty object. If there are no operators to start with on the destination server, your Compare command fails automatically.

Reply:
You will get this result:
![image](https://user-images.githubusercontent.com/11204251/53445221-ae80ba80-39d5-11e9-9c7a-e547c1123870.png)

Reply:
I'll work up an example of an alternate method that should include less overhead (improve timing, that compare command has overhead associated with it).